### PR TITLE
scenario/T39012

### PR DESCRIPTION
**Pull request by mabalderramav**

### CODE REVIEW

File: IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    /// <summary>
+    /// This class handles the CreateAccountReceip Page.
+    /// </summary>
+    public class CreateAccountReceiptPage : BasePage
+    {
+        [IDTFindBy(File = "CreateAccountReceiptLocators", Locator = "LblPinNamber")]
+        private IWebElement LblPinNumber;
+
+        /// <summary>
+        /// Verifys the data Basic Receipt in the CreateAccountReceip Page.
```
Comment (angy113) [inline IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs]: Miss spelling "Verifies" instead Verifys

File: IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        /// </summary>
+        /// <param name="dictionary">The <see cref="Dictionary"/> to verify.</param>
+        /// <returns>If true when assert all is true.</returns>
```
Comment (angy113) [inline IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs]: True if all asserts are true

File: IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        public bool VerifyBasicReceiptData(Dictionary<PinlessDial, string> dictionary)
+        {
+            bool result = false;
+            new List<PinlessDial>(dictionary.Keys).ForEach(key => result = this.GetStrategyStepMap(dictionary)[key].Invoke());
+            return result;
+        }
+
+        /// <summary>
+        /// Verify if is pin number is displayed.
```
Comment (angy113) [inline IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs]: Verifies if pin number is displayed

File: IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        /// </summary>
+        /// <param name="pinNumber">then pin number to verify.</param>
+        /// <returns>Ture if is pin number is displayed.</returns>
```
Comment (angy113) [inline IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs]: True if pin number is displayed.

File: IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        {
+            bool result = false;
+            new List<PinlessDial>(dictionary.Keys).ForEach(key => result = this.GetStrategyStepMap(dictionary)[key].Invoke());
+            return result;
+        }
+
+        /// <summary>
+        /// Verify if is pin number is displayed.
+        /// </summary>
+        /// <param name="pinNumber">then pin number to verify.</param>
```
Comment (angy113) [inline IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs]: The pin number to verify.

File: IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+        /// <returns>Ture if is pin number is displayed.</returns>
+        public bool IsPinNumberDisplayed(string pinNumber)
+        {
+            return this.LblPinNumber.GetAttribute("text").Equals(pinNumber);
+        }
+
+        /// <summary>
+        /// Obtains the map verifiys by the functions to verify the <see cref="CreateAccountReceiptPage"/>.
```
Comment (angy113) [inline IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs]: Obtains the map with the functions to verify the ..

File: IDT.Pinless/Main/UI/Pages/PinlessDial.cs [hunk 7]
```diff
@@ -0,0 +1,11 @@
+﻿// <copyright file="PinlessDial.cs" company="IDT">
+// Copyright (c) IDT. All rights reserved.
+// </copyright>
+
+namespace IDT.Pinless.Main.UI.Pages
+{
+    public enum PinlessDial
```
Comment (angy113) [inline IDT.Pinless/Main/UI/Pages/PinlessDial.cs]: Document class

File: IDT.Pinless/Main/UI/Pages/PinlessDial.cs [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+    {
+        PinNumber
```
Comment (angy113) [inline IDT.Pinless/Main/UI/Pages/PinlessDial.cs]: Document enum

File: IDT.Pinless/Main/Utils/Util.cs [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+// Copyright (c) IDT. All rights reserved.
+// </copyright>
+
+namespace IDT.Pinless.Main.Utils
+{
+    using System.Collections.Generic;
+    using TechTalk.SpecFlow;
+
+    /// <summary>
+    /// This class handles to method commons.
```
Comment (angy113) [inline IDT.Pinless/Main/Utils/Util.cs]: This class handles common methods

File: IDT.Pinless/Main/Utils/Util.cs [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+    /// </summary>
+    public static class Util
+    {
+        /// <summary>
+        /// Conteverts to Table to Dictionary.
```
Comment (angy113) [inline IDT.Pinless/Main/Utils/Util.cs]: Converts from table to dictionary

File: IDT.Pinless/Test/UI/Feature/br_ account.feature [hunk 11]
```diff
@@ -16,4 +16,11 @@
...
 			And I click on Back button
 		Then verify that Back button works ok for Pinless
-	
+
+	Scenario: T39012 - Verify that Basic Pinless creation successful
+		When I search for "(591) 608-9515" number in BR Account
+			And I create Basic plan in "Your Plans" section
+			And I click on Create Account button
+		Then Verify that Basic Pinless creation successful
+		| Field     | value          |
```
Comment (angy113) [inline IDT.Pinless/Test/UI/Feature/br_ account.feature]: value to Pascal case "Value"

File: IDT.Pinless/Test/UI/StepsDefinitions/ConfirmCreateAccountSteps.cs [hunk 12]
```diff
@@ -10,11 +10,24 @@ namespace IDT.Pinless.Test.UI.StepsDefinitions
     [Binding]
     public class ConfirmCreateAccountSteps
     {
+        private ConfirmCreateAccountPage confirmCreateAccountPage;
```
Comment (angy113) [inline IDT.Pinless/Test/UI/StepsDefinitions/ConfirmCreateAccountSteps.cs]: Follow naming conventions

File: IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs (hunk 1 above)
Comment (mabalderramav) [reply IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs]: Done

File: IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs (hunk 2 above)
Comment (mabalderramav) [reply IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs]: Done

File: IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs (hunk 3 above)
Comment (mabalderramav) [reply IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs]: Done

File: IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs (hunk 5 above)
Comment (mabalderramav) [reply IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs]: Done

File: IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs (hunk 4 above)
Comment (mabalderramav) [reply IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs]: Done

File: IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs (hunk 6 above)
Comment (mabalderramav) [reply IDT.Pinless/Main/UI/Pages/CreateAccountReceiptPage.cs]: Done

File: IDT.Pinless/Main/UI/Pages/PinlessDial.cs (hunk 7 above)
Comment (mabalderramav) [reply IDT.Pinless/Main/UI/Pages/PinlessDial.cs]: Done

File: IDT.Pinless/Main/UI/Pages/PinlessDial.cs (hunk 8 above)
Comment (mabalderramav) [reply IDT.Pinless/Main/UI/Pages/PinlessDial.cs]: Done

File: IDT.Pinless/Main/Utils/Util.cs (hunk 9 above)
Comment (mabalderramav) [reply IDT.Pinless/Main/Utils/Util.cs]: Done

File: IDT.Pinless/Main/Utils/Util.cs (hunk 10 above)
Comment (mabalderramav) [reply IDT.Pinless/Main/Utils/Util.cs]: Done

File: IDT.Pinless/Test/UI/Feature/br_ account.feature (hunk 11 above)
Comment (mabalderramav) [reply IDT.Pinless/Test/UI/Feature/br_ account.feature]: Done

File: IDT.Pinless/Test/UI/StepsDefinitions/ConfirmCreateAccountSteps.cs (hunk 12 above)
Comment (mabalderramav) [reply IDT.Pinless/Test/UI/StepsDefinitions/ConfirmCreateAccountSteps.cs]: Done

**Pull request merged**